Fix table on azure-storage-limits.md

Nican opened the pull request:
Table is not rendered correctly on https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#storage-limits 
The bottom half of the table is broken.

azurecla commented:
Hi __@jeannt__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jeannt). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Nican committed:
Fix table on azure-storage-limits.md

Table is not rendered correctly on https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#storage-limits 
The bottom half of the table is broken.
